mlflow/protos/databricks_managed_catalog_messages.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto2";
+
+package mlflow.managedcatalog;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "com.databricks.api.proto.managedcatalog";
+option java_generate_equals_and_hash = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+message TableInfo {
+
+  // Output-only fields (modified to include only what mlflow needs):
+
+  // [Create,Update:IGN BROWSABLE] Full name of Table, in form of <catalog_name>.<schema_name>.<table_name>
+  optional string full_name = 15;
+  // [Create,Update:IGN] Id of the table.
+  optional string table_id = 22;
+}
+
+message GetTable {
+  // Required. Full name of the Table (from URL).
+  optional string full_name_arg = 1;
+
+  // True if delta metadata should be included in the response.
+  optional bool include_delta_metadata = 3;
+
+  // Optional. Whether to omit the columns of the table from the response or not.
+  optional bool omit_columns = 5;
+
+  // Optional. Whether to omit the properties of the table from the response or not.
+  optional bool omit_properties = 6;
+
+  // Optional. Whether to omit the constraints of the table from the response or not.
+  optional bool omit_constraints = 7;
+
+  // Optional. Whether to omit the dependencies of the table from the response or not.
+  optional bool omit_dependencies = 8;
+
+  // Optional. When Specified, BROWSE privilege will be checked and returning browsable metadata when applicable
+  optional bool include_browse = 9;
+
+  // Optional. Whether to include information of shallow clones or not if this is base table.
+  optional bool include_shallow_clones = 10;
```

Comment:
Minor: do we need this? If the value is unspecified --> defaults to `false`, seems we'd just exclude shallow clone info, which we don't need (since we only need table ID)

Reply:
So these were low cost flags to add (i.e. I didn't need to port over an entirely new proto object to support them).  We can remove them and add them back in if/when we need them if you want this to be as minimal as possible.